tests/testutils.py

```diff
@@ -40,6 +44,8 @@ def tearDown(self):
         if bokeh:
             Store.renderers['bokeh'] = bokeh.BokehRenderer.instance()
         OutputSettings.options =  OrderedDict(OutputSettings.defaults.items())
+        for renderer in Store.renderers.values():
+            renderer.comm_manager = CommManager
```

Comment:
Why does ``comm_manager`` need to be set explicitly here? Isn't it the default?

Reply:
Various tests run the notebook extension which registers the Jupyter comms.

Reply:
Ok. And those won't be available in this context. Seems fine.